addOns/graphql/src/main/javahelp/org/zaproxy/addon/graphql/resources/help/contents/alerts.html

```diff
@@ -22,6 +22,11 @@ <h1>GraphQL Alerts</h1>
         <td>This alert is raised when the GraphQL implementation used by the server is identified. It utilises
             fingerprinting techniques adapted from the tool <a href="https://github.com/dolevf/graphw00f">graphw00f</a>.
 </table>
+
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/tree/main/addOns/graphql/src/main/java/org/zaproxy/addon/graphql/ExtensionGraphQl.java">ExtensionGraphQl.java</a>
```

Comment:
This is not the latest code, it should be `GraphQlParser` and `GraphQlFingerprinter` though it might be better to add to the table instead.